Library/Core/UnoCore/Targets/Android/Uno/Graphics/GLHelper.cpp

```diff
@@ -240,7 +240,7 @@ void GLHelper::_setEGLConfig(bool forPBuffer)
         eglGetConfigAttrib(_eglDisplay, configs[i], EGL_SAMPLES, &samples);
 
         if (samples >= cs && depth >= cd && buffer >= cb &&
-            samples <= samples && r <= colorBits.R && g <= colorBits.G && b <= colorBits.B && a <= colorBits.A)
+            r <= colorBits.R && g <= colorBits.G && b <= colorBits.B && a <= colorBits.A)
```

Comment:
Wow, this is actually a bug. There's two variables with the same name here, I bet the intention was to compare them.

That being said, I don't think we want to default to 4xMSAA any more, so maybe we should just get rid of that variable as well?

Anyway, this all seems a bit out-of-place here. I guess you noticed this due to some new warning when upgrading? Either way, I think this would deserve its own PR, unless it cause problems for this one without.

Reply:
I noticed this because of warnings introduced by the new tools, yes.

Another PR coming up! :)